### PR TITLE
add message to CreateOfficeRequestValidator

### PR DESCRIPTION
**Pull request by Cheephot**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/893

### CODE REVIEW

**Comment by Cheephot** [comment]
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/885

File: src/CompanyService.Validation/Office/CreateOfficeRequestValidator.cs [hunk 1]
```diff
@@ -4,15 +4,18 @@
...
+        .NotEmpty().WithMessage("Name must not be empty.");
 
-            RuleFor(request => request.Address)
-                .NotEmpty();
-        }
+      RuleFor(request => request.City.Trim())
+        .NotEmpty().WithMessage("City must not be empty.");
+
+      RuleFor(request => request.Address.Trim())
+        .NotEmpty().WithMessage("Address must not be empty.");
```
Comment (MariyaSin) [inline src/CompanyService.Validation/Office/CreateOfficeRequestValidator.cs:18]: add trim to mapper

Comment (Cheephot) [reply src/CompanyService.Validation/Office/CreateOfficeRequestValidator.cs:18]: done

File: src/CompanyService.Validation/Office/CreateOfficeRequestValidator.cs [hunk 2]
```diff
@@ -4,15 +4,18 @@
...
-    public class CreateOfficeRequestValidator : AbstractValidator<CreateOfficeRequest>, ICreateOfficeRequestValidator
+  public class CreateOfficeRequestValidator : AbstractValidator<CreateOfficeRequest>, ICreateOfficeRequestValidator
+  {
+    public CreateOfficeRequestValidator()
     {
-        public CreateOfficeRequestValidator()
-        {
-            RuleFor(request => request.City)
-                .NotEmpty();
+      RuleFor(request => request.Name.Trim())
```
Comment (BelehovEgor) [inline src/CompanyService.Validation/Office/CreateOfficeRequestValidator.cs:11]: if name is null you will have nullRefExc

**Review by VLiamin — APPROVED** [review]

File: src/CompanyService.Validation/Office/CreateOfficeRequestValidator.cs (hunk 2 above)
Comment (Cheephot) [reply src/CompanyService.Validation/Office/CreateOfficeRequestValidator.cs:11]: i fixed it in this PR
https://github.com/LT-Students/DigitalOffice-CompanyService/pull/86

**Pull request closed without merging**